pipelines/main/platforms/build_linux.arches

```diff
@@ -3,13 +3,13 @@
 linux      x86_64-linux-gnu          .             x86_64         x86_64         CFLAGS=-Werror,CXXFLAGS=-Werror                                         .             v5.8          582b5d44bbc24d2a58ab609c5a520b3fd102e504
 linux      x86_64-linux-gnuassert    .             x86_64         x86_64         FORCE_ASSERTIONS=1,LLVM_ASSERTIONS=1,CFLAGS=-Werror,CXXFLAGS=-Werror    .             v5.8          582b5d44bbc24d2a58ab609c5a520b3fd102e504
 # linux    aarch64-linux-gnu         .             aarch64        aarch64        .                                                                       .             ----          ----------------------------------------
```

Comment:
Did this line survive for a reason?

Reply:
Yes. I wanted to ask you - do we expect that building `aarch64-linux-gnu` will always succeed?

I put the tester for `aarch64-linux-gnu` in the "allowed to fail" list, but I wasn't sure about the builder.

Reply:
Cool, so let's plan to keep the builder as "normal", and the tester as "allowed to fail"?